core/frontend/src/IModelConnection.ts

```diff
@@ -157,6 +157,12 @@ export abstract class IModelConnection extends IModel {
    */
   public readonly onClose = new BeEvent<(_imodel: IModelConnection) => void>();
 
+  /** Event called immediately after *this* IModelConnection has its displayed extents expanded.
+   * @note This event is called only for this IModelConnection.
+   * @beta
```

Comment:
Should this API be marked external (beta), or left internal only?

Reply:
Internal. The concept of a "displayed extents" for an iModel doesn't make sense. It applies to all spatial views and it never shrinks. I would like to remove it in the future.